agent/util-scripts/pbench-make-result-tb

```diff
@@ -194,7 +194,7 @@ fi
 tarballmd5="${tarball}.md5.check"
 # We need to calculate the md5 sum in the temp directory
 # in order to get the filename right.
-pushd $(dirname ${tarball}) > /dev/null 2>&1
+pushd $(dirname ${tarball}) > /dev/null
 md5sum "$(basename ${tarball})" > "${tarballmd5}"
```

Comment:
This is still broken if a relative pathname is specified for `target-dir` as is done in #2360: we are *in* `./tardir` and we try save the md5 into `.tardir/<mumble>.tar.xz.md5.check`.

Reply:
Good catch!

One option would be to inflict `realpath(1)` on the input so that we know that we have an absolute, existing path with no weird components.  The other, as you've suggested, would be to use `basename(1)` to get rid of the path part.

Now...should one of us take over this PR and push it through, or do we want to wait for Peter's return?  (I suspect that we would rather not wait on getting the next 0.69 release out, and we would prefer to have this update in it.)

Reply:
OK, I think this is fixed, now.